var/spack/repos/builtin/packages/hypre/package.py

```diff
@@ -374,8 +374,10 @@ def install(self, spec, prefix):
             make("install")
             if spec.satisfies("+gptune"):
                 make("test")
-                self.run_test("mkdir", options=["-p", self.prefix.bin])
-                self.run_test("cp", options=["test/ij", self.prefix.bin + "/."])
+                mkdir = which("mkdir")
+                mkdir("-p", self.prefix.bin)
+                cp = which("cp")
+                cp("test/ij", self.prefix.bin + "/.")
```

Comment:
Probably want permissions set correctly, so better to use `install()` (see docs at https://spack.readthedocs.io/en/latest/llnl.util.html#llnl.util.filesystem.install):

```suggestion
                install(join_path("test", "ij"), self.prefix.bin)
```